testkit/appkit/block_resp.go

```diff
@@ -0,0 +1,82 @@
+package appkit
+
+import "time"
+
+/*
+This code section is auto-generated by this tool:
+https://mholt.github.io/json-to-go/
+
+This struct is based on the JSON response we get from
+the REST API from core's/app's node
+Please do not modify this struct manually.
+*/
+type BlockResp struct {
+	Jsonrpc string `json:"jsonrpc"`
+	ID      int    `json:"id"`
+	Result  struct {
+		BlockID struct {
+			Hash  string `json:"hash"`
+			Parts struct {
+				Total int    `json:"total"`
+				Hash  string `json:"hash"`
+			} `json:"parts"`
+		} `json:"block_id"`
+		Block struct {
+			Header struct {
+				Version struct {
+					Block string `json:"block"`
+					App   string `json:"app"`
+				} `json:"version"`
+				ChainID     string    `json:"chain_id"`
+				Height      string    `json:"height"`
+				Time        time.Time `json:"time"`
+				LastBlockID struct {
+					Hash  string `json:"hash"`
+					Parts struct {
+						Total int    `json:"total"`
+						Hash  string `json:"hash"`
+					} `json:"parts"`
+				} `json:"last_block_id"`
+				LastCommitHash     string `json:"last_commit_hash"`
+				DataHash           string `json:"data_hash"`
+				ValidatorsHash     string `json:"validators_hash"`
+				NextValidatorsHash string `json:"next_validators_hash"`
+				ConsensusHash      string `json:"consensus_hash"`
+				AppHash            string `json:"app_hash"`
+				LastResultsHash    string `json:"last_results_hash"`
+				EvidenceHash       string `json:"evidence_hash"`
+				ProposerAddress    string `json:"proposer_address"`
+			} `json:"header"`
+			Data struct {
+				Txs      []string `json:"txs"`
+				Evidence struct {
+					Evidence []interface{} `json:"evidence"`
+				} `json:"evidence"`
+				Msgs struct {
+					Msgs []struct {
+						NamespaceID string `json:"NamespaceID"`
+						Data        string `json:"Data"`
+					} `json:"msgs"`
+				} `json:"msgs"`
+				SquareSize string `json:"square_size"`
+			} `json:"data"`
+			LastCommit struct {
+				Height  string `json:"height"`
+				Round   int    `json:"round"`
+				BlockID struct {
+					Hash  string `json:"hash"`
+					Parts struct {
+						Total int    `json:"total"`
+						Hash  string `json:"hash"`
+					} `json:"parts"`
+				} `json:"block_id"`
+				Signatures []struct {
+					BlockIDFlag      int       `json:"block_id_flag"`
+					ValidatorAddress string    `json:"validator_address"`
+					Timestamp        time.Time `json:"timestamp"`
+					Signature        string    `json:"signature"`
+				} `json:"signatures"`
+			} `json:"last_commit"`
+		} `json:"block"`
+	} `json:"result"`
+}
```

Comment:
The whole file can be deleted after https://github.com/celestiaorg/test-infra/pull/27/files#r915969992 is adressed.

Reply:
Done. Thanks again for the hint!